src/api/app/views/webui2/webui/search/_advanced_search.html.haml

```diff
@@ -0,0 +1,19 @@
+.form-group
+  .card.mt-2.collapse#advanced-container
+    .card-body
+      %label Search in:
+      .form-group
+        .custom-control.custom-checkbox
+          = check_box_tag('name', 1, params[:name].nil? || params[:name] == '1', class: 'custom-control-input')
```

Comment:
It would be great to use speaking names here. But I realized that would require some refactoring and validation of `:classes` in the FullTextSearch model, which we should do anyway.

Still, I wanted to pitch the idea:-)